antispam_bee.php

```diff
@@ -1,7 +1,7 @@
 <?php
 /*
 Plugin Name: Antispam Bee
-Description: Easy and extremely productive spam-fighting plugin with many sophisticated solutions. Includes protection against trackback spam.
+Description: Easy and extremely productive spam-fighting plugin with many sophisticated solutions. Includes protection against trackback spam and privacy hints.
```

Comment:
This reads like `provides protection against [...] privacy hints`... Maybe swap around?